src/app.js

```diff
@@ -1,21 +1,9 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const { moveFile } = require('./moveFile');
+
+const args = process.argv.slice(2);
 
-module.exports = sum;
+if (args[0] === 'mv' && args.length === 3) {
```

Comment:
I am not sure whether you get mv in `argv,` maybe because of this your program does nothing